protobufs/canvas_generated/messages/events.proto

```diff
@@ -771,6 +771,7 @@ enum EventType {
   // VITALS_COMMAND__POST_EXECUTE_ACTION = 52011;
 
   PATIENT_CHART__CONDITIONS = 100000;
+  CLAIM__CONDITIONS = 101000;
```

Comment:
I think these should be named more specifically. They fire when these are being processed in a graphql resolver, right?

Reply:
Not this one; This one fires right after the serializer serializes data. :\